core/src/main/java/tech/tablesaw/io/string/DataFramePrinter.java

```diff
@@ -189,22 +190,25 @@ private String tableName(Relation frame, int width) {
             int i;
             for (i = 0; i < Math.ceil((double) rowCount / 2); i++) {
                 for (int j = 0; j < colCount; j++) {
-                    data[i][j] = frame.getString(i, j);
+                    Column<?> col = frame.column(j);
+                    data[i][j] = (col.size() > i ? col.getString(i) : "?");
```

Comment:
I'd probably use "X" or "" rather than "?"

Reply:
Can we have it print whatever it prints when the column is the correct size but contains a missing  value?  I believe calling getUnformattedString() handles that.

Reply:
I think you'd want to differentiate between empty and non-existent though. I think I'd rather focus on how to avoid getting the table into this state rather than fixing `toString` though

Reply:
> I think you'd want to differentiate between empty and non-existent though. 

I don't think there is a practical difference between empty and non-existent in this context. An empty string is just a convenient representation of "no value". It's the only datatype we suport that can even represent an empty value that's not a null value, I think consistency argues for dealing only in present or not-present (i.e. missing).

>  I think I'd rather focus on how to avoid getting the table into this state rather than fixing toString though

As @hallvard pointed out, since toString() is useful for debugging, we should try to ensure it doesn't blow up.

Reply:
I think we agree that a missing value (present, but using the missing value marker) is perfectly ok, while a too short column is a bug. That's why the output should be different, to help discover that something is wrong. It also seems we agree that toString shouldn't crash, so what remains is agreeing on what to show to indicate the missing-because-the-column-is-too-short situation. I prefer "?" since it's less likely to be a real value, but "x" is also fine. Perhaps we even can use 💣(bomb emoji unicode character)?

Reply:
Got it. Bomb would be nice, but lets go with "?"